src/algorithms/ctmrg.jl

```diff
@@ -384,7 +423,11 @@ function grow_env_left(peps, Pl, Pr, C_sw, C_nw, T_s, T_w, T_n)
     return C_sw′, C_nw′, T_w′
 end
 
-# Compute norm of the entire CMTRG enviroment
+"""
+    LinearAlgebra.norm(peps::InfinitePEPS, env::CTMRGEnv)
+
+Compute the norm of a PEPS contracted with a CTM environment.
+"""
```

Comment:
I'm not sure about this docstring, in principle it's already kind of inferred from the function in LinearAlgebra, and this will make this show up for any query of the LinearAlgebra.norm docstring.
I do like adding it in the documentation pages!

Reply:
Sure, I agree! However, I'm not sure how you would add the docstring only to the documentation pages while not making it available to any docstring query. Would you mind committing this?